playground/mocks/data/idp/idx/authenticator-enroll-duo.json

```diff
@@ -0,0 +1,194 @@
+                    "$.currentAuthenticator"
+                ],
+                "href": "http://localhost:3000/idp/idx/challenge/answer",
+                "method": "POST",
+                "produces": "application/ion+json; okta-version=1.0.0",
+                "value": [
+                    {
+                        "name": "credentials",
+                        "type": "object",
+                        "form": {
+                            "value": [
+                                {
+                                    "name": "signatureData",
+                                    "label": "Assertion From Duo",
+                                    "required": true,
+                                    "visible": false
+                                }
+                            ]
+                        },
+                        "required": true
+                    },
+                    {
+                        "name": "stateHandle",
+                        "required": true,
+                        "value": "02o1Im1DuPAqf5-TzR953e15dHuV619FapqRfTBzPv",
+                        "visible": false,
+                        "mutable": false
+                    }
+                ],
+                "accepts": "application/json; okta-version=1.0.0"
+            },
+            {
+                "rel": [
+                    "create-form"
+                ],
+                "name": "select-authenticator-enroll",
+                "href": "http://localhost:3000/idp/idx/credential/enroll",
+                "method": "POST",
+                "produces": "application/ion+json; okta-version=1.0.0",
+                "value": [
+                    {
+                        "name": "authenticator",
+                        "type": "object",
+                        "options": [
+                            {
+                                "label": "Duo Security",
+                                "value": {
+                                    "form": {
+                                        "value": [
+                                            {
+                                                "name": "id",
+                                                "required": true,
+                                                "value": "aut32kl92UF8kfE4E0g4",
+                                                "mutable": false
+                                            },
+                                            {
+                                                "name": "methodType",
+                                                "required": false,
+                                                "value": "idp",
+                                                "mutable": false
+                                            }
+                                        ]
+                                    }
+                                },
+                                "relatesTo": "$.authenticators.value[0]"
```

Comment:
Future suggestion - would love to see basic linting for these `.json` files to ensure all fields/values are present based on some schema.

This would allow us to more-or-less ignore these files when they're added to source control!

Reply:
Created https://oktainc.atlassian.net/browse/OKTA-369482 to follow up